src/lib.rs

```diff
@@ -240,22 +265,22 @@ fn create_proxied_request<B>(
 
         request
             .headers_mut()
-            .insert(&*TE_HEADER, HeaderValue::from_static("trailers"));
+            .insert(te_header(), HeaderValue::from_static("trailers"));
```

Comment:
Hey. I just realized that maybe this HeaderValue (and the one below for `UPGRADE`) should also use OnceCell?

Reply:
Yes, you're right, I meant OnceLock. I'll probably make this change later.